reference/toolstacks/python/README.md

```diff
@@ -0,0 +1,174 @@
+# Python
+
+Platform.sh supports deploying Python applications, your application can use
+WSGI-based (Gunicorn / uWSGI) application server, Tornado, Twisted, or Python 3.5+ asyncio server.
+
+## Supported versions
+
+* 2.7
+* 3.5
+
+## WSGI-based configuration
+
+In this example, we use Gunicorn to run our WSGI application.
+Configure the `.platform.app.yaml` file with a few key settings
+as listed below, a complete example is included at the end of this section.
+
+1. Specify the `type` as `python:2.7` (other versions you may use are listed above).
+2. Build your application with the build hook.
+   Assuming you have your pip dependencies stored in `requirements.txt` and
+   a `setup.py` at the root of your application folder to execute build steps:
+
+   ```yaml
+   hooks:
+     build: |
+       pip install -r requirements.txt
```

Comment:
See https://github.com/platformsh/platformsh-docs/pull/335/files/5c536872f1b2722bbfbd88b18b100d5afdf4e5b9#diff-e99730c25ef2aec5db351e1a9aa5de29R138 :) We will add more "server" dependencies